ui/org.eclipse.pde.core/src/org/eclipse/pde/internal/core/builders/JarManifestErrorReporter.java

```diff
@@ -138,11 +138,21 @@ protected void parseManifest(IDocument document, IProgressMonitor monitor) {
 					return;
 				}
 				// parse
+				String rest = document.get().substring(lineInfo.getOffset() + lineInfo.getLength()).trim();
```

Comment:
Please don't do that. This does eager reading almost all of the the manifest on every line!

Reply:
fixed it, changed the logic to avoid eager reading.